### ISSUE TITLE
Skip descriptor rows

### ISSUE THREAD

**Issue by dgfitch** [uwmadison-chm/masterfile#10]
See https://github.com/uwmadison-chm/scorify/issues/16

If scorify gets an option to keep descriptor rows around, we also need to make masterfile know about this.

If the "id" column is not a number, we ignore that row entirely.

It should probably get an optional setting that specifies a regex for ids.

It should also check and make sure that only rows 2-3 or so are getting skipped. Any later row that gets skipped is a BIG RED FLAG.

**Comment by njvack** [comment]
Yeah, we could have a --max_skip_rows option to `extract_masterfile_data` and default it to 3
